src/date/format-properties.js

```diff
@@ -208,6 +209,56 @@ return function( pattern, cldr ) {
 
 			// Zone
 			case "z":
+
+				// z...zzz: "{shortRegion}", eg. "PST" or "PDT".
+				// zzzz: "{regionName} {Standard Time}" or "{regionName} {Daylight Time}",
+				// eg. "Pacific Standard Time" or "Pacific Daylight Time".
+				if ( properties.tzId ) {
```

Comment:
We can leave it as is for now, but I have the following comment: who will determine the timezone name is actually date using an imaginary call like `date.getTimezoneName()`. This allows the case where `options.timeZone` is `undefined` that defaults to using local time.